File: 2.server/src/main/java/ru/vitaliy/petrov/server/forms/requests/CarAccidentWitnessUpdateRequest.java
package ru.vitaliy.petrov.server.forms.requests;

import lombok.Data;

@Data
public class CarAccidentWitnessUpdateRequest {
    private final String updatedWitnessFullName;
    private final String updatedWitnessResidentialAddress;
    private final String updatedWitnessPhoneNumber;
}

File: client/app/src/main/java/com/petrov/vitaliy/caraccidentapp/domain/models/requests/accident/CarAccidentWitnessUpdateRequest.kt
package com.petrov.vitaliy.caraccidentapp.domain.models.requests.accident

data class CarAccidentWitnessUpdateRequest(
    val updatedWitnessFullName: String,
    val updatedWitnessResidentialAddress: String,
    val updatedWitnessPhoneNumber: String,
    val witnessID: Long
)
